docs/api.rst

```diff
@@ -556,7 +556,15 @@ Extended handlers
     When True requires SSL/TLS to be established on the data channel. This
     means the user will have to issue PROT before PASV or PORT (default
     ``False``).
+	
+  .. data:: client_certfile
 
+    The path of the certificate to check the client certificate against. 
+    When provided, only allowing clients with a valid certificate to connect 
+    to the server (default ``None``).
+
+     .. versionadded:: 1.5.3    
+	
```

Comment:
Thanks for advice. The certificates client side and server side need are the basically same, but for client side, private key is needed. To my understanding, it's a bit similar to a "reversed" SSL/TLS certificate validation.